index.html

```diff
@@ -177,48 +200,57 @@ <h3>Profesional Art <br />Printing Data</h3>
       </div>
     </section>
     <section class="about-me">
-      <h1>About me</h1>
-      <p>
-        Hello I’m a software developer! I can help you build a product , feature
-        or website Look through some of my work and experience! If you like what
-        you see and have a project you need coded, don’t hesitate to contact me.
-      </p>
-      <a href="#" class="button">Get my resume</a>
+      <div class="illustration-about-me">
+        <h1>About me</h1>
+        <p>
```

Comment:
- [ ] I see 😃 that you used the h1 tag twice in the page ,so please try to use the h just once in the page this is for the semantics and SEO ,so you can use hierarchy headings like h2, h3 ... to be friendly to the semantics 😊

Reply:
@osamaashraf6 I have been trying so hard to make it aligned but unfortunately, it doesn't work